### ISSUE TITLE
Can we offer nightly pip packages? 

### ISSUE THREAD

**Issue by yaox12** [tensorflow/federated#345]
Recently I encounter a problem with the evaluation function in tensorflow federated,  as described [here](https://stackoverflow.com/questions/55510877/problem-with-evaluation-function-in-tensorflow-federated).

The answer below says it has been fixed in #309 and will be included in the next release.
I cannot wait for it and thus try to build the pip package from source. 
The package built by me works fine on CPU but fails on GPU, and I have no idea what went wrong.

So I hope you can offer official nightly built pip packages.
Thanks.

**Comment by michaelreneer** [comment]
@yaox12 Thanks for the feedback, publishing a nightly pip package is on our minds as well. It's good to know there would be demand. It also sounds like you did try and build the GPU pip package? Can you share your error message? Maybe I can help unblock you there short term.

**Comment by yaox12** [comment]
@michaelreneer It is exactly the same error as described in #361.
I have run the TFF tutorial code in [Colab](https://colab.research.google.com/drive/1joRyTabr9Euvj7eDJmTWE67jnpJmk6nM) (It is editable for convenience).
It works well when the runtime is set to CPU but occurs the above error using GPU.

**Comment by michaelreneer** [comment]
@yaox12 the GPU build is failing right now.

Duplicate of #361
